### PR TITLE
feat: added tests

### PR DESCRIPTION
**Pull request by amitojsingh**

### CODE REVIEW

File: src/components/ssa/ssa.test.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import Ssa from 'components/ssa';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+
+jest.mock('react-i18next', () => ({
+  useTranslation: (): { t: (key: string) => string } => ({
+    t: (key: string): string => key, // Mock the translation function to return the key
+  }),
+}));
+describe('Ssa component', () => {
```
Comment (MySmilingTurban) [inline src/components/ssa/ssa.test.tsx:14]: nit: add empty line

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]
one minor fix requested

**Comment by MySmilingTurban** [comment]
@amitojsingh veerji, shouldn't all test be in single folder named tests?

**Comment by amitojsingh** [comment]
@MySmilingTurban veer ji It's considered best practice to store unit test files adjacent to their corresponding source code files. This arrangement simplifies locating the relevant tests for each component of the application.

**Review by MySmilingTurban — APPROVED** [review]
Looks good to me!

**Pull request merged**